changelog.txt

```diff
@@ -1,4 +1,13 @@
-Version ??, June 1, 2020
+Version 3.5.0, July 1, 2020
+
+Renamed Signature Checker annotations:
```

Comment:
BinaryNameOrPrimitiveType, DotSeparatedIdentifiersOrPrimitiveType, and IdentifierOrPrimitiveType were also added right?

Reply:
I added them and some more text to the changelog.